cypress/plugins/index.js

```diff
@@ -1,4 +1,4 @@
-/// <reference types="cypress" />
+// / <reference types="cypress" />
```

Comment:
Is this extra space a typo?

Reply:
For some reason my linter was complaining about this bit, but it was fine the way it was before, so now I changed it back.